sparkbox/safe-focus
# Update dependencies with security issues

This PR updates the dependencies with security issues and removes JSDOM from the list of dependencies. 

## Summary
Includes updates for Mocha.

## Validation
1. Checkout `master`.
1. Run `rm -rf node_modules && npm i`.
1. Run `npm audit`
- [x] Confirm vulnerabilities are listed.
1. Checkout `chore--security-updates`.
1. Run `rm -rf node_modules && npm i`.
1. Run `npm audit`
- [x] Confirm no vulnerabilities are listed.
